src/Term/EntityTerms.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Wikibase\DataModel\Term;
+
+use Comparable;
+use InvalidArgumentException;
+
+/**
+ * Imutable value object.
+ *
+ * @since 3.1.0
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class EntityTerms implements Comparable {
+
+	/**
+	 * @var string
+	 */
+	private $languageCode;
+
+	/**
+	 * @var Term
+	 */
+	private $label;
+
+	/**
+	 * @var Term
+	 */
+	private $description;
+
+	/**
+	 * @var AliasGroup
+	 */
+	private $aliases;
+
+	/**
+	 * @param string $languageCode
+	 * @param Term|null $label
+	 * @param Term|null $description
+	 * @param AliasGroup|null $aliases
+	 *
+	 * @throws InvalidArgumentException
+	 */
+	public function __construct(
```

Comment:
PHPMD is correct, and this demonstrates quite nicely how a metric such as NPath is merely a useful tool, not a replacement for understanding the context.